package.json

```diff
@@ -157,6 +157,10 @@
         "language": "java",
         "path": "./snippets/java.json"
       },
+      {
+        "language": "java",
```

Comment:
I'm a big believer in "opt-in" on snippets. Perhaps you could change the language to be `java-testing`. 

This accomplishes two things: 
- Doesn't overload the users snippets if they don't want these particular testing snippets
- Users could then extend the snippets similar to this: 
```lua
require("luasnip").filetype_extend("java", { "java-testing" })
```

Reply:
Agree, this opt-in is a great feature, I will update to `java-testing`.

Thank you for the review =)

Reply:
Awesome! Thanks again :sunglasses: